src/molecules/formfields/SelectField/index.js

```diff
@@ -4,15 +4,24 @@ import classnames from 'classnames';
 import Tooltip from 'atoms/Tooltip';
 
 import { renderOption, renderPlaceholder } from 'utils/fieldUtils';
-
-import styles from 'molecules/formfields/shared/formfields.module.scss';
+import styles from './select_field.module.scss';
 
 function renderSelectOptions(options) {
   if (!options) return null;
 
   return options.map(renderOption);
 }
 
+const renderToolTip = (message) => {
+  if (!message) return null;
```

Comment:
@patrickkim Will putting this conditional here cause the functions that we use to process children to fail because it's injecting a null child instead of simply being skipped in the `{ message && renderToolTip(message) }` pattern?